cert/cert_test.go

```diff
@@ -219,6 +219,13 @@ func (certSuite) TestNewClientCertRSASize(c *gc.C) {
 	}
 }
 
+func (certSuite) TestComputePublicKey(c *gc.C) {
+	computedPublicKey, err := cert.ComputePublicKey(caCertPEM)
+	if err == nil {
```

Comment:
actually you want to c.Assert(err, gc.ErrorIsNil), if err is not nil this should fail too.